assets/components/svgs/newCreditCard.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+
+// The generic credit card logo used on checkouts.
+export default function SvgCreditCard() {
+
+  return (
+    <svg className="svg-credit-card" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 14" preserveAspectRatio="xMinYMid">
```

Comment:
Possibly worth naming this component "svg-new-credit-card"? It shouldn't matter, but this name is already taken by the component in `creditCard.jsx`.